shallowblue_rllib.py

```diff
@@ -321,10 +298,13 @@ def get_observation(self, world_state):
                     obs = np.rot90(obs, k=2, axes=(1, 2))
                 elif yaw == 90:
                     obs = np.rot90(obs, k=3, axes=(1, 2))
+
+                allow_break_action = observations['LineOfSight']['type'] == 'diamond_ore' or \
```

Comment:
kolby added line of sight stuff, which is more effective than doing allow_break_action from the grid